docs/pages/Workspace.md

```diff
@@ -0,0 +1,125 @@
+* Must be no longer than 50 characters
+* Must contain lowercase characters only
+* Must contain alphanumeric characters, dash, underscore and period only
+    - Exception: The `@` and `/` characters are allowed at certain positions as
+      explained below
+* Must start with an alphabetic character or an `@` character
+* If it starts with an `@` character, it must contain exactly one
+  forward slash `/`
+    - This is aligned with the npm concept for package scopes, for example `@org/lib.name`
+
+## Dependency Management
+
+UI5 Workspace configurations allow to influence the dependency resolution when working with a UI5 project.
+
+!!! example
+    ```yaml
+    specVersion: workspace/1.0
+    metadata:
+        name: dolphin
+	dependencyManagement:
+   		resolutions:
+       		- path: ../light.library
+       		- path: ../heavy.library
+       		- path: ../test.library
+    ```
+
+### Resolutions
+
+```yaml
+dependencyManagement:
+   	resolutions:
+       	- path: ../test.library
+```
+
+Resolution paths will be used by UI5 Tooling to look for project dependencies. Any dependencies found via these paths will be used over the ones found with the regular dependency resolution mechanism.
+
+Paths must point to a directory containing a `package.json`. UI5 Tooling will attempt to find a UI5 project here. However, if the `package.json` contains a [`workspaces`](https://docs.npmjs.com/cli/v8/using-npm/workspaces), or equivalently, `ui5.workspaces` configuration (which overrules the first), UI5 Tooling will resolve the workspace first and attempt to find UI5 projects in the configured "npm workspaces". This is commonly used in mono-repos. An example can be found in the [OpenUI5 repository](https://github.com/SAP/openui5/blob/b4267488e5d3546de4cd9577ccac4208482d71e0/package.json#L130-L132).
+
+Paths must be written in POSIX (i.e. using only forward slashes `/` as path segment separators) and must be **relative to the workspace configuration file**. Absolute paths, or paths relative to the home directory (`~`), are not allowed. This is to ensure that workspace configuration files are easy to share and to reduce the chance of them exposing sensitive information like user names or large directory hierarchies. Symbolic links are followed.
+
+Note that this configuration only affects the resolution of dependencies which have already been found during the regular dependency resolution process of a project. For example, if a workspace resolution path resolves to a project that would otherwise not be part of the dependency tree of the current root project, it will not be added to the dependency tree. Also, transitive dependencies of resolved projects are not being followed.
+
+## Workspace Specification Versions
+A workspace configuration must define a Specification Version by setting the `specVersion` property. UI5 Tooling uses this information to detect whether the currently installed version is compatible with a workspace's configuration.
```

Comment:
```suggestion
A workspace configuration must define a specification version by setting the `specVersion` property. UI5 Tooling uses this information to detect whether the currently installed version is compatible with a workspace's configuration.
```

Reply:
"Specification Version" ist nur dann ein proper noun, wenn es von einer Versionsnummer gefolgt wird. Wenn mal allgemein über "a specification version" redet, wird es klein geschrieben. (Btw, UI5 Tooling ist ein SAP brand name.)

Reply:
Actually I have to correct myself here. Günter just informed me that "Specification Version" shall only be used like this if it is followed by a version. So for example "Specification Version 3.0". In all other cases it shall be lowercased. Like in this case 👍